src/main/java/seedu/todo/controllers/concerns/Renderer.java

```diff
@@ -14,6 +14,22 @@
 public class Renderer {
     
     /**
+     * Renders an error message in both the console and the input field, leave null or empty string if not needed.
+     * 
+     * @param consoleMessage       Message to be rendered in the console
+     * @param consoleInputValue    Value to display in the input field
+     */
+    public static void showConsoleError(String consoleMessage, String consoleInputValue) {
```

Comment:
Hmm, yeah I'm not sure exactly what disambiguation messages you wanna show. Preferably you should be showing more detailed what went wrong in the message, the MESSAGE_DISAMBIGUATION is just a placeholder until then.